linkid_to_patid.py

```diff
@@ -133,17 +142,19 @@ def translate_linkids(args):
             source_name=source_metadata_filename,
             linkage_name=args.linkszip,
         )
-        if len(metadata_issues) == 0:
-            write_patid_links(args)
-        else:
+        if len(metadata_issues) > 0:
             print(
-                f"ERROR: Inconsistencies found in "
-                f"source metadata file {args.sourcefile}"
+                f"{'ERROR' if args.force else 'WARNING'}: "
```

Comment:
I know I was the one to suggest this but as I try it out now, I see I got it backwards. If not force mode it should be an error, if force mode it should be a warning. I'll make that change